src/components/PipelineRun/PipelineRunListView/PipelineRunListRow.tsx

```diff
@@ -115,6 +131,23 @@ const BasePipelineRunListRow: React.FC<React.PropsWithChildren<BasePipelineRunLi
           '-'
         )}
       </TableData>
+      <TableData className={pipelineRunTableColumnClasses.triggeredBy}>
+        {gitProvider === 'github' ? (
+          <GithubIcon className="git-icon" />
+        ) : gitProvider === 'gitlab' ? (
+          <GitlabIcon className="git-icon" />
+        ) : null}
+
+        {eventType ? (
+          <ExternalLink
+            href={`https://${gitProvider}.com/${repoOrg}/${repoURL}#${prNumber}`}
+            text={EventTypeLabel[eventType] ?? '-'}
```

Comment:
for pull requests
text should be in format `(icon) <org-name>/<repo-name>#<pr-number>`

for push event text should be commit id linking to either github commit or konflux commit details page.
Please get UX approval on this.